src/tests/13_Block.test.ts

```diff
@@ -0,0 +1,167 @@
+import { expect } from 'chai'
+import UserService from '../db/service/UserService'
+import UserInfoService from '../db/service/UserInfoService'
+import BlockService from '../modules/blocking/BlockService'
+import UserCollection, { UserDocument } from '../db/models/user.model'
+import UserInfoCollection, { UserInfo } from '../db/models/userInfo.model'
+
+import { cleanupMockedUserInfo, createAndTestUserInfo } from './2_UserInfo.test'
+import { generateRandomPassword } from '../utils/utils'
+
+export interface MockedUser {
+	mockedUser: UserDocument;
+	mockedInfo: UserInfo;
+}
+
+describe('Block service can', () => {
+	let blockService: BlockService
+	let mockedBlocker: MockedUser
+	let mockedUser: MockedUser
+	let userService: UserService
+	let userInfoService: UserInfoService
+
+	describe('setup', () => {
+		it('instantiate the service class', () => {
+			blockService = new BlockService()
+			expect(blockService).not.equal(undefined)
+		})
+
+		it('create prerequesite services', () => {
+			userService = new UserService(UserCollection)
+			userInfoService = new UserInfoService(UserInfoCollection)
+			expect(userService).not.equal(undefined)
+			expect(userInfoService).not.equal(undefined)
+		})
+
+		it('create two mocked users', async () => {
+			const mockedUserInput = {
+				createUserInput: {
+					email: 'mockedUser@email.com',
+					password: await generateRandomPassword(),
+					username: 'mockedUser2'
+				},
+				createInfoInput: {
+					firstname: 'Mocked',
+					lastname: 'User',
+					email: 'mockedUser@email.com',
+					username: 'mockedUser2'
+				}
```

Comment:
You can wrap them into a function and use it in Block and Follow test to reduce code duplication (sonar check failed because of this duplication)

Reply:
Done we'll see if the code duplication is under 3%